css/qmwp.css

```diff
@@ -595,4 +595,24 @@ body.qmwp-lightbox-visible {
 =====================================*/
 .qmwp-linked-accounts {
     font-weight: bold;
-}
\ No newline at end of file
+}
+
+table.available-shortcodes {
+    border-collapse: collapse;
+}
+
+table.available-shortcodes td {
+    padding: 10px 0;
+    border-width: 1px 0;
+    border-color: #000;
+    border-style: solid;
+}
+
+table.available-shortcodes tr:last-of-type td {
+    border: none;
+}
+
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Element (td.shortcode) is overqualified, just use .shortcode without element name.](https://www.codacy.com/app/m_3/QuantiModo-WordPress-Plugin/file/1049789523/issues/source?bid=2836188&fileBranchId=2883147#l615)